docs/source/workflow_inputs.rst

```diff
@@ -399,7 +399,7 @@ For these simulations:
 Whole SFA/MF Buildings
 **********************
 
-As of v1.7.0, OpenStudio-HPXML can model whole SFA/MF buildings in a single combined simulation.
+Alternatively, OpenStudio-HPXML can model whole SFA/MF buildings in a single combined simulation.
```

Comment:
Consider a footnote on Notes for `WholeSFAorMFBuildingSimulation` that when false, the individual dwelling unit will be specified using a translator argument?

Reply:
Good idea. I addressed this and did some related improvements. Let me know if you have any more feedback.